installPhase.sh

```diff
@@ -15,11 +15,19 @@ else
     VERSION_APPENDIX="-NOGIT"
 fi
 
+# Extract version from Cargo.toml using toml2json
+PACKAGE_VERSION=$(toml2json < Cargo.toml | jq -r .package.version)
```

Comment:
release-please ensures that we bump these in parallel so we don't need to read from `cargo.toml`.